python/ray/data/dataset.py

```diff
@@ -2823,13 +2914,15 @@ def write_mongo(
         collection: str,
         ray_remote_args: Dict[str, Any] = None,
     ) -> None:
-        """Write the dataset to a MongoDB datasource.
+        """Write the :class:`~ray.data.Dataset` to a MongoDB datbase.
 
-        This is only supported for datasets convertible to Arrow records.
-        To control the number of parallel write tasks, use :meth:`Dataset.repartition``
-        before calling this method.
+        This is only supported for datasets convertible to PyArrow tables.
```

Comment:
For consistency within the docstring 
```suggestion
        This is only supported for datasets convertible to pyarrow tables.
```